src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -35,18 +39,15 @@ export interface EslintStrategy {
 	/** Get engine that strategy supports */
 	getEngine(): ENGINE;
 
-	/** Get eslint engine to use for scanning. */
-	getRunConfig(engineOptions: Map<string, string>): Promise<Record<string, any>>;
+	/** Get config options for eslint engine to use for scanning. */
+	getRunOptions(engineOptions: Map<string, string>): Promise<ESLint.Options>;
 
 	/** Get languages supported by engine */
 	getLanguages(): string[];
 
 	/** After applying target patterns, last chance to filter any unsupported files */
 	filterUnsupportedPaths(paths: string[]): string[];
 
-	/** Filters out any rules that should be excluded from the catalog */
-	filterDisallowedRules(rulesByName: Map<string,ESRule>): Map<string,ESRule>;
```

Comment:
`Strategy`s still implement this method, but it's only ever called within the classes themselves. So the method has been removed from the abstract class contract.